ComAmazonawsKms/Makefile

```diff
@@ -13,3 +13,13 @@ MAX_RESOURCE_COUNT=10000000
 AWS_SDK_CMD := --aws-sdk
 STD_LIBRARY=StandardLibrary
 SMITHY_DEPS=model
+
+format_net:
+	pushd runtimes/net && dotnet format && popd
```

Comment:
question: do we need/want to do this for ddb as well?

Reply:
Oh, I didn't mean to add this... this was probably a merge issue.
I'll resolve this to whatever's on main.

Reply:
Removed, I believe this was moved into the general SmithyDafnyMakefile.